client/coral-admin/src/routes/Moderation/components/Comment.js

```diff
@@ -63,6 +72,16 @@ const Comment = ({
                 </span>
               )
             }
+            {
+              minimal && typeof selected === 'boolean' && typeof toggleSelect === 'function' && (
+                <input
+                  className={styles.bulkSelectInput}
+                  type='checkbox'
+                  value={comment.id}
+                  checked={selected}
+                  onChange={(e) => toggleSelect(e.target.value, e.target.checked)} />
```

Comment:
It would be great to have this defined once, instead on every render. This would be more performant and also allow future optimisations like marking this component `pure`.